.rubocop.yml

```diff
@@ -512,3 +512,21 @@ Style/Encoding:
 # so once RE-5565 is resolved, this should be enabled.
 Metrics/PerceivedComplexity:
   Enabled: true
+
+# This defaults to a much smaller number, but correcting the offending methods
+# will take a bit of work, so let's just not write anything more complex than
+# what we already have!
+Metrics/AbcSize:
+  Max: 45
+
+# We have a few modules that are much larger than the default limit even when
+# we don't count the comments
+Metrics/ModuleLength:
+    CountComments: false
+    Max: 200
+
+# I have no idea why, but it's complaining that we aren't using octal notation
+# correctly when specifying file permissions. I have no idea how to resolve the
+# complaint without disabling this cop
+Style/NumericLiteralPrefix:
+  Enabled: false
```

Comment:
[Here's the doc](https://github.com/bbatsov/ruby-style-guide#numeric-literal-prefixes)

I don't love this.